sapi/fpm/fpm/fpm_status.c

```diff
@@ -62,7 +62,8 @@ int fpm_status_export_to_zval(zval *status)
 
 	/* copy the scoreboard not to bother other processes */
 	scoreboard = *scoreboard_p;
-	struct fpm_scoreboard_proc_s procs[scoreboard.nprocs];
+	struct fpm_scoreboard_proc_s *procs = safe_emalloc(
```

Comment:
Note: you can use do_alloca which can either do alloca or a heap allocation; but I guess this is not a performance-critical piece of code?

Reply:
yeah exactly and it should also use per request memory (emalloc) for this to count to mem limit. The stack is also not used for status api calls...

Reply:
FWIW, do_alloca does use `emalloc`; but sure.

Reply:
ah ok, but still not much point to put this on stack.